### PR TITLE
Support Podman network mode

### PR DESCRIPTION
**Pull request by maturar**
Issues: #8 

# Definition of Done

The PR shall be merged **only if all of following items were checked.** In case an item is not applicable as described, please provide a short explanation right after the item in question. Separate the explanation by a semicolon and write it as bold text like **; requirements are handled in issue #xyz**:

- [x] documentation of all modules `/*/doc/swdesign` is up-to-date
- [x] requirements are up-to-date and requirements for new features have been added and mapped to source and test - will be updated in #1 
- [x] conform to [unit verification strategy](https://eclipse-ankaios.github.io/ankaios/main/development/unit-verification/) - will be done in #1 (we need integration tests for this change).

### CODE REVIEW

**Review by lingnoi — APPROVED** [review]
👍 LGTM

**Review by isbm — COMMENTED** [review]
A little remark from me though.

File: doc/docs/reference/startup-configuration.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ The runtime configuration for the `podman` runtime is specified as followed:
 | mounts | array of [Mount](#mount)| no | List of mounts
 | ports | array of [Mapping](#mapping) | no | List of ports to be exposed
 | remove | boolean | no | Specify whether the container shall be removed after exited
+| networkMode | string | no | Set the network mode for the container (like `bridge`, `host`, `none`)
```
[the remark this answers is not in the document]
Comment (lingnoi) [reply doc/docs/reference/startup-configuration.md:66]: @isbm: Maybe one aspect you haven't considered and this is the fact that this `runtimeConfig` is the config for Podman runtime and shall behave the same way like the config of podman which allows also networkMode = "none".
So I suggest we should leave it like it is.

[the remark this answers is not in the document]
Comment (maturar) [reply doc/docs/reference/startup-configuration.md:66]: I have added a note to the documentation to make the situation (with the runtime configuration) clear.

[the remark this answers is not in the document]
Comment (isbm) [reply doc/docs/reference/startup-configuration.md:66]: @maturar ah, right that's just a proxypass: `networkMode="none"` means "Create a network without an interface".

**Review by lingnoi — APPROVED** [review]

**Pull request merged**